website/blog/2023-05-30-0.72-metro-package-exports-symlinks.md

```diff
@@ -0,0 +1,173 @@
+
+#### More ECMAScript support in Hermes
+
+Support for the following specifications in Hermes.
+
+- prototype.at support for [Array](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at), [TypedArray](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/at) and [String](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/at). See [#823](https://github.com/facebook/hermes/issues/823) ([ebe2](https://github.com/facebook/hermes/commit/ebe2915ac386a6b73dec39c2af4ac7063e68cd99)).
+- Implement [well-formed JSON.stringify](https://github.com/tc39/proposal-well-formed-stringify) ([d41d](https://github.com/facebook/hermes/commit/d41decf244aa814b1e58827a9de982f3b71667de)) to prevent ill-formed Unicode strings
+- Implement [AggregateError](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/AggregateError) ([9b25](https://github.com/facebook/hermes/commit/9b25a2530eb515f6c4fbd397ae290b6c97c049b2)) that represents several errors wrapped in a single error. Useful for multiple errors like from [`Promise.any()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/any) when all promises passed to it reject.
+
+For users on JSC, these features are already available.
+
+### Mitigation for TextInput ANRs on Android
+
+Earlier this year our team was alerted of frequent and persistent ANRs (application not reponsive) around multi-line TextInputs from a high volume of community reports. Investigating, we isolated the issue to devices using the Samsung soft keyboard with Grammarly enabled ([issue/35936](https://github.com/facebook/react-native/issues/35936)).
+
+Our best understanding is a correlation of using and re-formatting a large number of `AbsoluteSizeSpan`s. We were able to reproduce the issue outside of React Native, on a template Android application and found this issue also impacts the larger Android app ecosystem.
+
+We have reached out to Samsung to share our findings and hope it will be addressed by Samsung in a future update.
+
+As a mitigation for the React Native community, we have landed a change to reduce our dependency on `AbsoluteSizeSpans` in cases where they can be omitted. This is not a complete solution but lowers number of cases this ANR may occur.
+
+This mitigation is in 0.72 and has been back-ported to the following versions:
+
+- [0.71.7](https://github.com/facebook/react-native/releases/tag/v0.71.7)
+- [0.70.9](https://github.com/facebook/react-native/releases/tag/v0.70.9)
+- [0.69.10](https://github.com/facebook/react-native/releases/tag/v0.69.10)
+- [0.68.7](https://github.com/facebook/react-native/releases/tag/v0.68.7)
```

Comment:
```suggestion
This mitigation is in 0.72 and has been back-ported to the following versions:

- [0.71.7](https://github.com/facebook/react-native/releases/tag/v0.71.7)
- [0.70.9](https://github.com/facebook/react-native/releases/tag/v0.70.9)
- [0.69.10](https://github.com/facebook/react-native/releases/tag/v0.69.10)

While we have backported these fixes in all the versions within the [support window policy](https://github.com/reactwg/react-native-releases#releases-support-policy), and even went the extra mile by backporting to [0.68.7](https://github.com/facebook/react-native/releases/tag/v0.68.7), we want to stress how we recommend to every React Native developer to always upgrade to the latest stable available.
```

Reply:
I'm going to re-word this but the support window bit is a good point to add!